web/css/root.css

```diff
@@ -121,12 +122,74 @@ nav {
   flex-direction: column;
 }
 
-nav > a {
+.header-btn {
+  min-height: 28px;
+  width: 28px;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 0;
+  padding: 0;
+  color: var(--second-font-color);
+  background-color: var(--second-bg-color);
+  position: fixed;
+}
+
+.header-btn:not(:disabled):hover {
+  background-color: var(--second-bg-color);
+}
+
+.header-btn-arrow-left {
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+  width: 0px;
+  margin: auto;
+  transform: rotate(135deg);
+  -webkit-transform: rotate(135deg);
+  color: var(--second-font-color);
+}
+
+.header-btn-arrow-right {
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+  width: 0px;
+  margin: auto;
+  transform: rotate(-45deg);
+  -webkit-transform: rotate(-45deg);
+  color: var(--second-font-color);
+}
+
+.header-btn-toggle {
+  outline: none;
+}
+
+/* show header button on small screens */
+
+@media screen and (min-width: 0px) and (max-width: 480px) {
+  .header-btn-open .header-btn-closed {
+    display: block;
+  }
+}
+
+/* hide header button elsewhere */
+
+@media screen and (min-width: 601px) {
```

Comment:
let's show the button for everything below 1024px, so let's change this to 1025px